tgrsite/templates/tgrsite/basecontent.html

```diff
@@ -104,6 +125,17 @@ <h1>{% block pagetitle %}{% endblock %}</h1>
     <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js"
             integrity="sha384-JjSmVgyd0p3pXB1rRibZUAYoIIy6OrQ6VrjIEaFf/nJGzIxFDsf4x0xIM+B07jRM"
             crossorigin="anonymous"></script>
+    {% if user.is_authenticated and not user.member.seen_tutorial %}
+    <script type="text/javascript">
+        $(document).ready(function(){
+            $('#tutorial-modal').modal();
+            $.get("/tutorial-done");
+        });
+    </script>
+    {% endif %}
+    <style>
```

Comment:
This probably wants to me inside the above if, and then manually re-included in the main tutorial page. Probably want this to be an overridable block too for that